frontApp/app/settings.tsx

```diff
@@ -0,0 +1,197 @@
+import { View, Text, Touchable, TouchableOpacity, StyleSheet, Image, ScrollView } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { Link, useRouter } from 'expo-router';
+import { Ionicons, MaterialIcons, MaterialCommunityIcons, Entypo } from '@expo/vector-icons';
+
+import { TextInput } from 'react-native-gesture-handler';
+
+const Settings = () => {
+    const router = useRouter();
+
+    return (
+        <View style={styles.container}>
+            
+            <View style={styles.header}>
+                <View style={styles.headerBtn}>
+                    <TouchableOpacity onPress={() => router.push("/register")} >
+                        <View>
+                            <Ionicons name="arrow-back-circle-outline" size={42} color={'#fff'} />
+                        </View>
+                    </TouchableOpacity>
+                </View>
+
+                <View style={styles.header2}>
+                    <View>
+                        <Text style={styles.headerText}>Settings</Text>
+                    </View>
+                </View>
+
+            </View>
+
+            <View>
+            <Text style={styles.settingsTitleText}>General</Text>
+                <TouchableOpacity style={styles.settingsBtn}>
+                    <View style={styles.settingsView}>
+                        <View style={styles.iconSettings}>
+                            <Ionicons name="person-circle-outline" size={32} color="#000" />
+                        </View>
+                        <Text style={styles.settingsBtnText}>Edit Profile</Text>
+                        <MaterialIcons name="arrow-forward-ios" size={32} color="#000" />
+                    </View>
+                </TouchableOpacity>
+            </View>
+
+            <View>
+            <Text style={styles.settingsTitleText}>Account</Text>
+                <TouchableOpacity style={styles.settingsBtn}>
+                    <View style={styles.settingsView}>
+                        
+                        <View style={styles.iconSettings}>
+                            <MaterialCommunityIcons name="cancel" size={32} color="#000" />
```

Comment:
Idk what is right or wrong but how do you feel about using different Icon packages? I think it might be ok but just asking for your opinion. Guess I'm ok as long as we are consistent.

Reply:
we are okay with using multiple imports for now. will discuss to condense later